db/migrate/20220430103855_create_users.rb

```diff
@@ -0,0 +1,8 @@
+class CreateUsers < ActiveRecord::Migration[6.1]
+  def change
+    create_table :users do |t|
+
+      t.timestamps
+    end
```

Comment:
-  [ ] There is no migration column added. Make sure you have proper migration created for user. 

![image](https://user-images.githubusercontent.com/58504655/166104223-e9d55f8b-4524-41ee-8e52-506c7a3ed275.png)

Reply:
thanks for the feedback @arslanbisharat but I think you didn't go through my PR  carefully you would have the migrations
to `add_name_to_user`,`add_photo_to_user` and `add_post_counter_to_user`

Reply:
I can see you have created individual migration for each property but this makes the database hard to read and manage. Please try to follow the comments given below.